CHANGES.rst

```diff
@@ -11,6 +11,11 @@
 
 - 100% branch coverage.
 
+- Make the C and Python `TimeStamp` objects behave more alike. The
+  Python version now produces the same `repr` and `.raw()` output as
+  the C version, and has the same hashcode. In addition, the Python
+  version is now supports ordering and equality like the C version.
+
```

Comment:
Sorry, I was unclear.  What I really meant was "two blank lines above the 4.0.8 version heading".

(This rule is not maintained consistently: e.g. there's only one blank line above 4.0.7, but two blank lines above 4.0.6, 4.0.5, and 4.0.4.  There *were* two blank lines above the 4.0.8 heading before your changes, which is why I commented.)